model/bindings.go

```diff
@@ -53,7 +48,7 @@ func CreateBindingsRancher(pathToYml string) BindingProperty {
 				log.Errorf("Error during Unmarshal for file %s : %v\n", dockerFile, err)
 			} else {
 				rawConfigDocker, err := preprocess.PreprocessServiceMap(rawConfigDocker)
-				utils.Convert(rawConfigDocker, &configMap)
+				_ = rawConfigDocker
```

Comment:
Please never type `_ = someVariable`.. it is never the correct solution to the unused variable compiler warning.